utils/rpms/daos.spec

```diff
@@ -129,7 +130,12 @@ Requires: %{name} = %{version}-%{release}
 Requires: %{name}-client = %{version}-%{release}
 Requires: spdk-tools
 Requires: ndctl
-Requires: ipmctl
+# needed to set PMem configuration goals in BIOS through control-plane
+%if 0%{?is_opensuse}
```

Comment:
If we are going to have a separate dependency for SLES/Leap, this should apply to SLES as well as Leap.  `%{is_opensuse}` won't match for SLES.

But I really don't think setting a max. version is the correct solution.  Providing a package that meets the EL7 requirements is the correct solution here otherwise we might start introducing distro-specific regressions due to tool version mis-matches.

Reply:
it would have been useful to have this feedback earlier in the PR cycle